crates/egui_demo_lib/src/demo/demo_app_windows.rs

```diff
@@ -409,11 +409,13 @@ mod tests {
             // Run the app for some more frames...
             harness.run_ok();
 
-            let mut options = SnapshotOptions::default();
             // The Bézier Curve demo needs a threshold of 2.1 to pass on linux
-            if name == "Bézier Curve" {
-                options.threshold = 2.1;
-            }
+            let threshold = OsThreshold::new().linux(2.1);
+            // Somehow on windows we need to allow 5 pixels of difference
+            let pixel_threshold = OsThreshold::new().windows(5).linux(5).macos(5);
```

Comment:
I think the `new` function should take the defaullt:

```suggestion
            let threshold = OsThreshold::new(0.0).linux(2.1);
            let pixel_threshold = OsThreshold::new(5);
```